podcasts/Bookmarks/Player/BookmarksPlayerTab.swift

```diff
@@ -1,44 +1,91 @@
 import SwiftUI
+import PocketCastsDataModel
+import PocketCastsUtils
 
 struct BookmarksPlayerTab: View {
+    @ObservedObject var viewModel: BookmarkListViewModel
     @EnvironmentObject var theme: Theme
 
+    @State private var showShadow = false
+
     var body: some View {
-        ZStack {
-            Color.blue
+        VStack(spacing: 0) {
+            headerView
+            divider
+            scrollView
+        }
+        .environmentObject(viewModel)
+        .padding(.bottom)
+        .background(theme.playerBackground01.ignoresSafeArea())
+    }
 
-            Text("Hello!")
-                .foregroundStyle(Color.white)
+    /// A static header view that displays the number of bookmarks and a ... more button
+    private var headerView: some View {
+        HStack {
+            Text(L10n.bookmarkCount(viewModel.bookmarkCount))
+                .foregroundStyle(theme.playerContrast02)
+                .font(size: 14, style: .subheadline)
+
+            Spacer()
+
+            Image("more").foregroundStyle(theme.playerContrast01).buttonize {
+                print("NOOP")
+            }
         }
+        .padding(.horizontal, Constants.padding)
+        .padding(.bottom, Constants.headerPadding)
     }
-}
 
-// MARK: - PlayerItemViewController Wrapper
+    private var scrollView: some View {
+        ZStack(alignment: .top) {
+            ScrollViewWithContentOffset {
+                LazyVStack(spacing: 0) {
+                    //   enumerated to get both the index and the bookmark to hide the last divider
+                    ForEach(Array(viewModel.bookmarks.enumerated()), id: \.element.id) { index, bookmark in
```

Comment:
You can achieve the same result and get rid of this `enumerated` and the `divider` logic if you add the divider on the top (before `BookmarkRow`) and remove the `body` `divider`.

Reply:
I actually ended up changing how this works in a future PR.

Reply:
Oh ok!